### ISSUE TITLE
Character Art

### ISSUE THREAD

**Issue by boxesofflowers** [Infinite-Dango-Studios/ganbatte#12]
Hello!
As far as I can tell, for each character we need four bases (neutral, excited, sad, angry), and each one of their outfits (which I think is four or more). If we make the outfits separately, maybe in layers like this: 
![example](https://cloud.githubusercontent.com/assets/11141654/6420534/ee91c134-be94-11e4-8ba6-acbdcc3f4ef7.png)

We could then save the file under the character name as a .psd (to preserve the layers) then we could make the combinations quickly, and they'd all be identical. This way, everything is organized. 

However, this would be much much easier if there were 
a) less emotions
b) less characters
c) no pose changes
d) less outfits

If we stick with amount of characters we have (16) assuming that they all have at least 3 bases and 3 outfits, this would be almost 200 different possible combinations. We'd need at least 48 bases and 144 different costumes. This is mainly because in the concept sketches, poses change with emotion for each character, and each character has a unique original base. If poses change, so do the way garments would fall on the figure, meaning another set of 3 or 4 costumes for each different emotion.

We could definitely pull this off, we'd just have to be very very precise, and only create what we'd need. I don't know if there's a full script anywhere, but if there is, character emotions and costumes should be listed with each line of dialogue.Ex. "Sally: What's going on?? (neutral, default)"

I'm going to get a head start on digitizing Jacoba's sketches for the bases, and I'll be uploading a color palette with each character, for consistency. 

That's pretty much all I have to say, I'll be posting what I finish in this issue.

**Comment by ExcaliburZero** [comment]
That file structure sounds good.

That is quite a lot of artwork to do. We should make sure to only do what we really need.

Currently the amount of the script that is coded in the game can be found in the "/game/" folder of the repo under the "day_01.rpy" and "day_02.rpy". The character images are commented into the files, in that they are in the files, but just aren't being run yet, as there are no character images so far. But if you look through those commented out show statements ("#show kosuke happy") it will list all of the on-screen character image changes. Some of the image names are somewhat preliminary, as I was unable to unify some of the character poses listed in the script by Sabre-Hecate.

Earlier I went through and got together a list of all of the background images so far (for image declarations), and I think I will do the same thing soon with character images. Once I do that, there will be a list in "/game/declarations.rpy" that will contain all of the character images that are currently in the coded script. I will post again on this issue once I do so.

**Comment by ExcaliburZero** [comment]
I've gone through and gotten a list of all of the character images in the day_01 script and put them in the declarations file ("/game/declarations.rpy") as image declarations. See 4f9eb65acca7add157a057fa53f888de2afe009d .

I've tried to unify much of them, but some of the the more unusual emotions may need to be merged or removed to lighten the load of the artwork.

**Comment by ExcaliburZero** [comment]
I have finished going through the day_02 script and updated the list of character images in the declarations file. See a109bc4a264b6f1f0a2e17e3d804bf45f364a00e .

**Comment by boxesofflowers** [comment]
This is incredible, oh my gosh thank you so much for that. Reading through, a lot of the emotions are unnecessary, most games in this genre only have two or three different versions of one character, it's a stretch to have so many variations. 
I'm going to try and digitize some of the sketches, I'll post them here in a bit.

**Comment by ExcaliburZero** [comment]
@boxesofflowers 
You're welcome.

Yeah, a lot of the emotions need to be looked over, merged or removed. I think @Sabre-Hecate might want to do that, since he's working on the writing.

Sounds good.

**Comment by juanitaylor** [comment]
this makes so much sense, actually. I don't want to cut out anything, but I realize that it's much more practical to do it this way. I would like to point out that the might change, especially since body language will help us portray characters. Also I made some poses for Misaki already and don't want them to go un-used xD

**Comment by boxesofflowers** [comment]
I'm working from your Misaki poses right now! I may have to change her proportions a bit, her torso and legs are really long, but yeah I suck at poses I'm definitely using your poses A+++++

**Comment by Sabre-Hecate** [comment]
Ahh, haha, I need to have fun, so I name his emotions funny things. 

Most of them can be boiled down to a few emotions. Evil and Smug are both Smirk. Ummm, what else...embarrassed is blush. Beaming is happy. Basically, if it sounds like it is happy, sad, angry, blush, or smirk, it probably IS.

**Comment by boxesofflowers** [comment]
those are still five emotions, if we wanted to do all different poses for each one, that might be too much, but for Smirk and Blush I think we could just change the facial expression of the default, that way only extreme emotions (happy, sad, angry) have large pose changes. That'll cut down on the number of bases we need :)

**Comment by boxesofflowers** [comment]
![misaki](https://cloud.githubusercontent.com/assets/11141654/6426769/7b6e1ba4-bf38-11e4-8dad-05e8eb76d638.png)
here's the base I have, I think I'm going to redo it, I'm not happy with the look.

**Comment by ExcaliburZero** [comment]
I'll try to go through with Sabre-Hecate and consolidate the extra character images later today.

**Comment by Sabre-Hecate** [comment]
I think we have a little bit of lost in translation going on here. 

When I say emotions, I literally just mean facial expressions. I don't mean expressions and poses. I should have made that more clear. Blush and smirk can definitely be the standard pose, as could some others, TBH. Like, Misaki needs a Tsun-tsun look (ideally) and a Yandere look, but both of those are just faces for standard pose. Feel free to decide what needs different poses at your discretion. Anger probably. Maybe sad. It's up to you. 

Have a :dango: for your troubles

**Comment by boxesofflowers** [comment]
@Sabre-Hecate that's good news! I just saw that in Jacoba's concept sketches she'd made each emotion a different pose, just changing the face would cut down A LOT of work

**Comment by Sabre-Hecate** [comment]
Ahhh, I'm glad this confusion got sorted. 

Some of the emotions can have different poses, depending on what the artists want, but not all of them need poses. Like I said, smirk and blush especially can be just face. Maybe other special faces like Tsun-Tsun and Yandere for Misaki. It'll be up to you guys.

**Comment by juanitaylor** [comment]
I think we can be a lot more expressive if we have different poses for the emotions, it will just take longer.
Excellent work on the base, by the way, @boxesofflowers !!!

**Comment by Sabre-Hecate** [comment]
That's true, @juanitaylor.... decide with @boxesofflowers it's up to you two....

**Comment by boxesofflowers** [comment]
I don't know if any of you have photoshop, but I've finished the Misaki nuetral base and costumes. Here are the images out of the file:
![misaki](https://cloud.githubusercontent.com/assets/11141654/6539809/85fb7b2e-c44d-11e4-937b-1c941940b2be.png)
I'll probably end up redoing the outfits I've drawn so far, they're not super flattering on the base. I want to make the body realistic but also I want her to look hot because yo she's hot, so, I'm conflicted.

**Comment by juanitaylor** [comment]
I finished more bases for neutral poses today, will post soon

I really think we need better communication, @boxesofflowers . It looks good so far, but i wouldn't like to think either of us is doing work that won't be used.

**Comment by boxesofflowers** [comment]
Are we showing the whole character on screen or just bellybutton up. Belly button up might look better, I could add more details and aestetically it might look nicer.

**Comment by boxesofflowers** [comment]
![image](https://cloud.githubusercontent.com/assets/11141654/6656909/92d07c64-cb0f-11e4-9a22-655cc1bf8c28.png)
every image i can find pretty much uses this format, I think it would make a lot more sense

**Comment by ExcaliburZero** [comment]
@boxesofflowers 
I think we will probably just be showing the characters' upper half. That would probably work well.

**Comment by boxesofflowers** [comment]
also, no artwork for the main character, correct?

**Comment by ExcaliburZero** [comment]
@boxesofflowers 
Yep, we will not be showing the main character on-screen, except for maybe in CG screens or other things like that.

**Comment by boxesofflowers** [comment]
alright, awesome!

**Comment by boxesofflowers** [comment]
![kosuke](https://cloud.githubusercontent.com/assets/11141654/6658333/9a602ba0-cb3c-11e4-8125-cb2f40edc055.png)
I tried Kosuke, I'm not a huge fan of this style, I'm going to redo it I think.

**Comment by Sabre-Hecate** [comment]
KATAWA SHOUJO! 

Anyway. 

KOSUKE~!!!

**Comment by boxesofflowers** [comment]
@Sabre-Hecate Do you like this face better?
![kosuke 2](https://cloud.githubusercontent.com/assets/11141654/6674474/4f77032a-cbf0-11e4-86cb-e35b709aab6f.png)

**Comment by Sabre-Hecate** [comment]
Hmm, it's a little hard to compare because I'm not an artist and one is finished while the other is not. I think I like the second better? But I'm not sure...? It's hard to judge...?

**Comment by boxesofflowers** [comment]
![kosuke 2](https://cloud.githubusercontent.com/assets/11141654/6698228/56ed8bdc-cccd-11e4-9999-56c5e731b2e8.png)
okay so this is my final design for kosuke, i hope you like it..:)

**Comment by boxesofflowers** [comment]
I'm going to color him very soon, as well as change facial expressions/mouth positions, but this is the final design I like (sort of)

**Comment by Sabre-Hecate** [comment]
OHMIGOD OHMIGOD YOU GUYS! HE LOOKS SO GOOD :fire:

**Comment by boxesofflowers** [comment]
@Sabre-Hecate I'm glad you like it! :)

**Comment by juanitaylor** [comment]
aaauuuurgh I feel terrible
y'all are forging ahead and I'm
nope
@boxesofflowers you're doing fantastic keep it up
I'm wondering what I should work on??? Why don't you tell me what you need, @boxesofflowers .

**Comment by boxesofflowers** [comment]
@juanitaylor hello friend!! backgrounds would be super duper, but if you wanna work on characters, I have your initial sketches but details and close ups and hair ideas and just anything you can think of. + backgrounds are super important and i suck at backgrounds :)

**Comment by boxesofflowers** [comment]
![misaki](https://cloud.githubusercontent.com/assets/11141654/6753734/abdaaa26-ceee-11e4-88ab-ae8a357b2be5.png)
this is a tentative final design for misaki??

**Comment by Sabre-Hecate** [comment]
WHOA. SHE'S HOT TOO. THEY'RE ALL HOT. :fire:  :fire: :fire:  ATTRACTIVE CHARACTER DESIGNS.

**Comment by Sabre-Hecate** [comment]
Hey, quick question. What do Yuuki's clothes look like again???? I'm deciding the details of her backstory. I know that her parents and lil sister Usagi were killed by  murderer. I'm thinking that the murderer uses a scythe for Battle Royale related reasons. Because, they're going to see her at a theater when they're watching Battle Royale and Mitsuko uses a scythe as her signature weapon and yeah. So anyway, point is, I need to know what her clothes look like. Because I kinda want her to have a scar. Like, she almost died, but lived. And it screwed her up, hence why she is so demon crazy and stuff. But, I don't want the scar visible. I want it to be something that Haruki discovers when getting intimate with her for a dramatic scene towards the beginning/middle of her route. Where should the scar be??

Note: Don't worry about a design for Usagi or Mr. and Ms. Akuma. They, like Haruki, will only be seen in special art scenes if at all.

**Comment by juanitaylor** [comment]
We discussed this on facebook, but I think you and I agreed that across her chest would be invisible when she's dressed. If @boxesofflowers wanted to change the design, that would be fine.
